packages/orbit-design-tokens/docs/SCHEMA.md

```diff
@@ -0,0 +1,276 @@
+# Schema of @kiwicom/orbit-design-tokens
+
+Orbit design tokens could be considered as a simple way to describe, document and use the visual language of Orbit design system, but there are multiple requirements that it needs to meet and support.
+
+Therefore, we use structured and standardized way Orbit design tokens are being created, stored and how we work with them.
+
+## Namespace / Object / Variant / Property
+
+In similarity with **Category / Type / Item** defined by style-dictionary we aim to have design tokens defined by some hierarchical tree structure.
+
+This hierarchical tree structure brings consistent approach how our design tokens are being defined in systematical way. Therefore, you will see that every definition of our design token will look like this:
+
+```json
+{
+  "component": {
+    "button": {
+      "large": {
+        "height": {
+          "type": "size",
+          "value": "{foundation.size.extra-large}"
+        }
+      }
+    }
+  }
+}
+```
+
+The above example is being translated with custom attribute transforms. For information how we work with `namespace.object.variant.property` hierarchical tree structure check transforms document.
+
+### Namespace
+
+In our design tokens an `namespace` is considered the top-level layer of design tokens that determines the intention of the token – where and how exactly it will be used.
+
+Possible values of the object level are:
+
+- The `foundation` is being used for the internal design tokens that are used as aliases for `global` and `component` design tokens, and/or interpretation of values that can be overwritten by custom theming,
+- the `global` is being used for the most common design tokens that interprets the fundamental visual language of the Orbit design system,
+- the `component` is being used for grouped design tokens that can be used for similar components, e.g. shared design token for the height of the buttons and/or inputs.
+
+### Object
+
+The `object` is an attribute to specify either the name of some component (component group) or [foundation object](#foundation-objects).
```

Comment:
Object here is not an object in classing meaning, above it's described the structure like: 

`## Namespace / Object / Variant / Property`

so for instance, in case of the size foundation: 

```
   "foundation": {
    "size": {
      "small": {
        "type": "size",
        "value": 16,
        "internal": true
      },
    .... 
```

where the foundation is the namespace and "size" is the Object that contains normal object

Reply:
feel free to suggest the alternative :) I also just moved that part from the old MR

Reply:
Ahh, understood. I got confused by `object` (the JS primitive) and the name `Object` (used in this tokens scope) 😅 I think it is acceptable as is, but I would personally prefer something like

```
The `object` attribute is used to specify either the name of some component (component group) or the [foundation object](#foundation-objects).
```

Just because I think it leaves no doubt between the two _objects_. But I will leave the decision for you.